src/modules/party/party.controller.ts

```diff
@@ -64,18 +66,40 @@ export class PartyController {
   })
   async createParty(
     @ValidatedArrayBody({ items: CreatePartyRequestItem })
-    createPartyDto: CreatePartyRequest,
+    createPartyDto: CreatePartyRequestDto,
     @Res({ passthrough: true }) res: Response,
   ): Promise<CreatePartyResponse> {
     const [newParty] = createPartyDto;
-    const partyIdentifierOfMatchingParty = await this.partyService.getPartyIdentifierBySearchText(newParty.alternateIdentifier);
+    const { alternateIdentifier, officerRiskDate: ratedDate } = newParty;
+
+    const partyIdentifierOfMatchingParty = await this.partyService.getPartyIdentifierBySearchText(alternateIdentifier);
 
     if (partyIdentifierOfMatchingParty) {
       res.status(HttpStatus.OK);
       return partyIdentifierOfMatchingParty;
     }
```

Comment:
Is it right that we don't try to create the external rating if the party already exists?